packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchorMenuItem.razor.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+public partial class NimbleAnchorMenuItem : ComponentBase
```

Comment:
Can we make a NimbleAnchorBase class like we did in Angular and Components so we can avoid duplicating all these properties and docs?

Reply:
Done.